task/dev1/printers/PrintValueToConsole.java

```
@@ -0,0 +1,31 @@
+package task.dev1.printers;
+
+import java.util.List;
+
+/**
+ * Print input value to console
+ */
+public class PrintValueToConsole {
+
+  /**
+   * Print object to console
+   * @param value - input object witch need to print
+   */
+  public void printObject(Object value) {
+    if (value != null) {
```

Ignoring error is a very-very-very bad practice